content/reference/templates/index.md

```diff
@@ -0,0 +1,56 @@
+# Templates
+
+BookingSync themes are made up of Liquid template files, each serving their own unique purpose. For example, `search.liquid` is used to display multiple rentals, and `rental.liquid` is used to show the details of a single rental.
+
+## Theme Structure
+
+A BookingSync theme comprises the following directories:
+
+* images
+* javascripts
+* layouts
+* snippets
+* stylesheets
+* templates
+
+### Images directory
+
+The `images` directory contains all the images used in the theme, including theme images and vendors images.
+
+### Javascripts directory
+
+The `javascripts` directory contains all the javascripts files used in the theme.
+
+You can write javascript using `javascript` or `coffeescript` langages.
+
+### Layouts directory
+
+The `layout` directory contains theme layout templates. All Liquid templates inside the templates folder are rendered inside the theme.liquid file.
+
+### Snippets directory
+
+The `snippets` directory contains all the theme's Liquid snippet files, which are bits of code that can be referenced in other templates of a theme.
+
+Use the `include` tag to load a snippet into your theme.
+
+### Stylesheets directory
+
+The `stylesheets` directory contains all the CSS files used in the theme.
+
+You can write CSS using `css` or `scss` langages.
+
+You can also use [Bourbon](http://bourbon.io/), a simple and lightweight mixin library for Sass.
```

Comment:
Bourbon is included in our default theme but is not forced, this should then be removed as theme dependent, not builtin.